leapp/snactor/__init__.py

```diff
@@ -68,6 +79,17 @@ def cli(args):
 
 
 def main():
+    profile_enabled = os.environ.get('LEAPP_CPROFILE', '0') == '1'
+    if profile_enabled:
+        pr = cProfile.Profile()
+        pr.enable()
     os.environ['LEAPP_HOSTNAME'] = socket.getfqdn()
     load_commands()
     cli.command.execute(version=_('snactor version {}').format(VERSION))
+    if profile_enabled:
+        pr.disable()
+        s = StringIO.StringIO()
```

Comment:
I am not totally sure, but this should be:
```suggestion
        s = StringIO()
```
right?

Reply:
yes!